### DISCUSSION TITLE
Comments: The past, present and future of virtualization

### DISCUSSION THREAD

**Discussion by jiayihu** [jiayihu/blog#64] [Announcements]
This issue is reserved for comments to [The past, present and future of virtualization](http://blog.jiayihu.net/the-past-present-and-future-of-virtualization/). Leave a comment below and it will be shown in the blog page.

**Comment by anBertoli** [comment]
Nice recap of virtualization strategies. Also, I appreciate references.

**Comment by jiayihu** [comment]
Why did you read the article 😂 bored in the house and checking my profile?

**Comment by anBertoli** [comment]
More or less 😂
